torch/csrc/autograd/autograd.cpp

```diff
@@ -103,7 +104,8 @@ variable_list run_backward(
           input.requires_grad(),
           "One of the differentiated Tensors does not require grad");
       if (!grad_fn) {
-        output_edges.emplace_back();
+        // See the python_engine's run_backward for an explanation
```

Comment:
nit: when we want to do these references, we usually name these like that:
https://github.com/pytorch/pytorch/blob/1c8d11c9e2ff64d92fb322c573d287588c26a718/torch/csrc/autograd/variable.h#L330

And reference them with that name (so that grepping for the note is easy)
https://github.com/pytorch/pytorch/blob/1c8d11c9e2ff64d92fb322c573d287588c26a718/torch/csrc/autograd/variable.h#L98

Reply:
Cool!